nifty-core/src/main/java/com/facebook/nifty/core/NettyServerTransport.java

```diff
@@ -131,6 +131,7 @@ public ChannelPipeline getPipeline()
                 SslServerConfiguration serverConfiguration = sslConfiguration.get();
                 if (serverConfiguration != null) {
                     SslHandler handler = serverConfiguration.createHandler();
+                    handler.setEnableRenegotiation(false);
```

Comment:
ya i should remove this from this particular diff. This is not needed for client support. I think this snuck in.